package.json

```diff
@@ -34,11 +34,13 @@
     "build:ui": "cd embark-ui && npm run build",
     "clean": "rimraf dist embark-*.tgz package embark-ui/build",
     "eslint": "eslint",
-    "install:core": "npm install",
-    "install:ui": "cd embark-ui && npm install",
+    "install:core": "yarn install",
+    "install:ui": "cd embark-ui && yarn install",
     "install_all": "npm-run-all install:*",
     "lint": "npm-run-all lint:*",
-    "lint:js": "eslint babel.config.js bin/embark embark-ui/src/ src/bin/ src/lib/",
+    "lint:js": "npm-run-all lint:js:*",
+    "lint:js:core": "eslint babel.config.js bin/embark src/bin/ src/lib/",
+    "lint:js:ui": "cd embark-ui && npm run lint",
```

Comment:
Not 100% sure why (yet) but I had to put the linting of `embark-ui/src` inside an embark-ui npm-script, otherwise there was a dependency resolution problem.